bin/run-ansible.sh

```diff
@@ -392,15 +403,23 @@ if [[ "${playbook}" = "server_and_nuc_deploy" ]]; then
     else
         aurora_inventory="ansible/inventory/server_and_nuc/${aurora_inventory}"
     fi
-    ansible_flags="${ansible_flags} --ask-vault-pass"
+    if [[ "${test_machine}" = "true" ]]; then
+        ansible_flags="${ansible_flags} --vault-password-file /home/$USER/vault.sh"
```

Comment:
How does the vault.sh file look like? Should we add an exmple file? or a comment describing the contents of such file?